### ISSUE TITLE
Enable debug log mode for Auth Lambda in production

### ISSUE THREAD

**Issue by MCatherine1994** [bcgov/nr-forests-access-management#1298]
**Describe the task**
In this case for debugging, we might want to update the log level from info to debug in production. Not sure if we need to change the level in code or how to config that. 

Some thought, refine during refinement session.
1. Migrate current `logging.config` file to either **json** (preferable, since python has built-in parser) or **yaml** for modern Python logging package recommended approach using: `logging.config.dictConfig`. 
ref: https://docs.python.org/3/library/logging.config.html

2. Define an environment variable (we define the name we like) for log level to be used (to dynamically set level during runtime).

3. Modify code in (auth lambda) `lambda_function.py` to configure logging with above (1.) configuration file and test if it is still good.

4. Figure out (still not too clear), modify code in (auth lambda) `lambda_function.py` to use the environment variable from (2.) to switch log level during or after logging configuration.
Perhaps can try this:
![Image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/e93aa1fc-aab8-40e8-b4a1-512bb5ed4d23)

5. Test locally if it can be dynamically switching the level. Possibly just run the `auth_function` tests with setting the new log level environment into `pytest.ini` before running tests and observe the output.

6. When deployed to AWS, at the AWS Lambda Function console, probably we can set this (I never try it before) and look into CouldWatch log after several user login tries:
![Image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/dfd1e5e6-c9b6-4b0d-a078-315ac14729dd)

**Acceptance Criteria**
- [x] Use an environment variable to determine log level, default to the level we're currently using (INFO)
- [x] Test in DEV environment

**Additional context**
- Add any other context about the task here.
- Or here

**Comment by ianliuwk1019** [comment]
This has been included in PR: https://github.com/bcgov/nr-forests-access-management/pull/1806.
Can be closed if PR is merged.

**Comment by ianliuwk1019** [comment]
Yes, this was incorporated in the past. I will close this ticket. @OlgaLiber2
